qutip/solver/heom/bofin_solvers.py

```diff
@@ -903,17 +903,11 @@ def steady_state(
             Specifies the the maximum number of iterative refinement steps that
             the MKL PARDISO solver performs.
 
-            For a complete description, see iparm(8) in
-            http://cali2.unilim.fr/intel-xe/mkl/mklman/GUID-264E311E-ACED-4D56-AC31-E9D3B11D1CBF.htm.
-
```

Comment:
We can link to https://www.intel.com/content/www/us/en/docs/onemkl/developer-reference-c/2023-0/pardiso-iparm-parameter.html instead.

Reply:
@hodgestar could you confirm the iparm index shift?

Reply:
It looks like this is a shift in the numbering. Perhaps in the earlier documentation the parameters started at 1 and now they start at 0. Let's assume that this is the case.